### PR TITLE
Adds fabulous new blanket sprite

### PR DESCRIPTION
**Pull request by HugoLuman**
Signed-off-by: HugoLuman <qaggeler@uci.edu>

backcape backcape2

:cl:
imageadd: added cosmic space blankie sprite
add: adds cosmic space blankie item for later inclusion in loot
/:cl:

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
As we talked earlier, it'd be neat to add a bedsheet to Lavaland or one of the adventure zones.

No point in adding a sprite that won't be used by the game, admins can upload sprites for events if they need to do that.

**Comment by ktoma36** [comment]
You know, we could add some more loot to find with interesting effects as well. I could see that bedsheet being sort of invisibility cloak that only works on space tiles

**Comment by HugoLuman** [comment]
Well, I have no idea how to code that, but feel free to help with that if you want

File: code/game/objects/structures/bedsheet_bin.dm [hunk 1]
```diff
@@ -222,6 +222,16 @@ LINEN BINS
 	item_color = "ian"
 	dream_messages = list("a dog", "a corgi", "woof", "bark", "arf")
 
+/obj/item/bedsheet/cosmos
+	name = "cosmic space blankie"
+	desc = "Made of the dreams of space children everywhere."
+	icon_state = "sheetcosmos"
+	item_color = "cosmos"
+	dream_messages = list("the infinite cosmos", "hanz zimmer music", "a flight through space", "the galaxy", "being fabulous", "shooting stars")
```
Comment (FlattestGuitar) [inline code/game/objects/structures/bedsheet_bin.dm]: a) it's "Hans"
b) names in English are capitalized >:C

File: code/game/objects/structures/bedsheet_bin.dm [hunk 2]
```diff
@@ -222,6 +222,16 @@ LINEN BINS
 	item_color = "ian"
 	dream_messages = list("a dog", "a corgi", "woof", "bark", "arf")
 
+/obj/item/bedsheet/cosmos
+	name = "cosmic space blankie"
```
Comment (FlattestGuitar) [inline code/game/objects/structures/bedsheet_bin.dm]: These are all called bedsheets, not blankets. I still don't know what these words mean in English, but I think it's better to stick to what's already there.

File: code/game/objects/structures/bedsheet_bin.dm [hunk 3]
```diff
@@ -222,6 +222,16 @@ LINEN BINS
 	item_color = "ian"
 	dream_messages = list("a dog", "a corgi", "woof", "bark", "arf")
 
+/obj/item/bedsheet/cosmos
+	name = "cosmic space blankie"
+	desc = "Made of the dreams of space children everywhere."
```
Comment (FlattestGuitar) [inline code/game/objects/structures/bedsheet_bin.dm]: Not something I wouldn't merge this over, but I think there's potential for a way cooler, more cosmic desc. I mean, look at that sprite. It's so beautiful...

**Pull request merged**